examples/demo-nft-module/queries.md

```diff
@@ -0,0 +1,3 @@
+curl -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"nft_getCollectionAddress","params":["sov1l6n2cku82yfqld30lanm2nfw43n2auc8clw7r5u5m6s7p8jrm4zqrr8r94","Test Collection"],"id":1}' http://127.0.0.1:12345
```

Comment:
Let's explain what each of the command does. If we decide to make `demo-not-module` a standard module then we could integrate it in `demo-rollup`  the same way as for bank.

Reply:
ah sorry, this particular file was checked in accidentally and i was just saving the curl commands there - the explanation for the queries is here https://github.com/Sovereign-Labs/sovereign-sdk/blob/aa75d9ca57e533feb837a671d89eb6eba17c84bb/examples/demo-nft-module/README.md#create-collection I can add more detail there as well and remove this file